torchinductor/lowering.py

```diff
@@ -3006,9 +3015,10 @@ def sum_(x, axis=None, keepdims=False, *, dtype=None):
 register_pointwise(aten.gt, type_promote=False, override_dtype=torch.bool)
 register_pointwise(aten.eq, type_promote=False, override_dtype=torch.bool)
 register_pointwise(aten.ne, type_promote=False, override_dtype=torch.bool)
-register_lowering(aten.__and__, type_promote=False)(
```

Comment:
Why remove `__and__`?

Reply:
It's not removed, it's moved one line down, to use `logical_and` definition.